cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -51,7 +52,8 @@ internal_data_port=40010
 # nodes that already in the cluster, unnecessary to be the nodes that were used to build the
 # initial cluster by start-node.sh(.bat). Several nodes will be picked randomly to send the
 # request, the number of nodes picked depends on the number of retries.
-seed_nodes=127.0.0.1:9003,127.0.0.1:9005,127.0.0.1:9007
+#seed_nodes=127.0.0.1:9003,127.0.0.1:9005,127.0.0.1:9007
+seed_nodes=127.0.0.1:9003
```

Comment:
Is it unfriendly for users to configure only one IP in distributed mode?
Why change the default parameter?

Reply:
Actually THIS is  "friendly".

What we want:
1. let user download the zip (iotdb-cluster.zip)
2. unzip the file
3. call `start-node.sh`
4. begin to have a try.

If we set 3 ips here, then the process is.... very very difficult for users:
1. let user download the zip (iotdb-cluster.zip)
2. unzip the file
3. copy the file into 3 parts
4. modiy `iotdb-cluster.properties` one by one (very hard, as they have to read the file to know "ah, in the first file, I need modify the parameter as 9003, and modify the rpc port and internal_data port as .... In the second file, I need ...."
5. call `start-node.sh`
6. begin to have a try.

It is a disaster for fresh man.

If you see other distributed systems, all their default setting is just one node.

Reply:
Sure, your consideration is reasonable.